submissions/EmmanuelIdiaghe/domManipulation/newsletter.js

```diff
@@ -0,0 +1,34 @@
+/*
+    Form Validation: This helps to ensure that only the correct input type is entered in the individual fields.
+*/
+// Declaration of variables
+let newsBtn    = document.getElementById('newsBtn');
+
+//Function to validate form
+function validateNews(event) {
+    let newsEmail  = document.getElementById('newsEmail'),
```

Comment:
I think you should declare all this variables outside of the function @emmanuelidiaghe